ingestion_tools/pyproject.toml

```diff
@@ -44,7 +53,7 @@ imageio = "^2.33.1"
 pytest = "^8.3.2"
 boto3-stubs = {extras = ["s3"], version = "^1.34.34"}
 mypy = "^1.8.0"
-cryoet-data-portal-neuroglancer = { git = "https://github.com/chanzuckerberg/cryoet-data-portal-neuroglancer.git", tag = "v1.0.0" }
+cryoet-data-portal-neuroglancer = { git = "https://github.com/chanzuckerberg/cryoet-data-portal-neuroglancer.git", tag = "v1.0.1" }
```

Comment:
Bump to patch version for fixing build issue in `cryoet-data-portal-neuroglancer`